go/vt/vtgate/engine/distinct.go

```diff
@@ -116,7 +115,7 @@ func (pt *probeTable) hashCodeForRow(inputRow sqltypes.Row) (evalengine.HashCode
 	code := evalengine.HashCode(17)
 	for i, checkCol := range pt.checkCols {
 		if i >= len(inputRow) {
-			return 0, vterrors.New(vtrpcpb.Code_INTERNAL, "distinct check colls is larger than its input row")
+			return 0, vterrors.VT13001("distinct checkCol is larger than its input row")
```

Comment:
`checkCol` here references a code level variable, but isn't it more clear to have a more human description of it?

Reply:
Fixed via [87e2f03](https://github.com/vitessio/vitess/pull/10738/commits/87e2f0356724e92c4b69f9cae25de67bc7a83fe5)